index.html

```diff
@@ -25,56 +30,112 @@
                 <a class="nav-link active" aria-current="page" href="#">Home</a>
               </li>
               <li class="nav-item">
-                <a class="nav-link" href="#">About</a>
-              </li>
-              <li class="nav-item">
-                <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
+                <a class="nav-link" href="pages/about.html">About</a>
               </li>
             </ul>
           </div>
-        </div>
       </nav>
-      <h1 class="col-sm-12 order-2">Super Sportcars Global Race 2021</h1>
-      <h2 class="col-sm-7 order-1">"Hello sharing World"</h2>
-      <p class="col-sm-12 order-3">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Expedita tempora pariatur provident minima qui aliquam dicta itaque veritatis! Repellendus tempora iusto perspiciatis ipsum maiores, adipisci numquam atque reiciendis illum error.</p>
-      <p class="col-sm-12 order-4">01/02/2021(sat) ~ 16(sun)</p>
-      <p class="col-sm-12 order-5">@ la recta de Azua, Azua, Dominican Republic</p>
+      <div class="col-12">
```

Comment:
- [ ] **The design of your hero section doesn't match the original. Please fix this.**

Original Design|Cloned Design
:-------------------:|:------------------:
![image](https://user-images.githubusercontent.com/59168430/124209357-03cdb900-db07-11eb-8929-eb79923195fb.png)|![image](https://user-images.githubusercontent.com/59168430/124209434-29f35900-db07-11eb-99ba-a3824e0fd4bc.png)


- The original design has some spacing on the left side of the text. The cloned design is missing the spacing. 
- The text hierarchy is a very important aspect of design. In the original design, the text is separated in the hierarchy using font sizes. The cloned design is missing that separation. For instance, the block of text below the subheading has a very large font size, or the text below the date also has a large font size.